app/src/index.scss

```diff
@@ -0,0 +1,23 @@
+@use 'styles/cssReset';
+@use 'styles/breakpoints';
+
+html {
+  font-size: 10px;
+
+  @media (max-width: breakpoints.$media-m-tablet) {
+    font-size: 8px;
+  }
+}
+
+body {
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen,
```

Comment:
I recommend creating variables with the font-family that we want to use in our application